server/server.js

```diff
@@ -41,6 +43,9 @@ app.use(
   })
 );
 app.use("/api/user", userRouter);
+app.use("/api/lectures", lectureRouter);
```

Comment:
To be consistent with the other routes I would use the singular: `app.use("/api/lecture", lectureRouter);`